weave

```diff
@@ -1131,8 +1131,14 @@ launch_proxy() {
     # when launching the weaveproxy container.
     docker_client_args $DOCKER_CLIENT_ARGS
     proxy_args "$@"
+    HOSTS_PATH=$(docker run --rm \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      --entrypoint /bin/sh \
+      $EXEC_IMAGE -c 'docker inspect -f {{.HostsPath}} $HOSTNAME')
```

Comment:
It took me a while to figure this out. `$HOSTNAME` gets resolved inside the container, right?